.coafile

```diff
@@ -14,6 +14,7 @@ bears = LineLengthBear
 
 [all.links]
 bears = InvalidLinkBear
+link_ignore_regex = (github\.com|\{|\$)
```

Comment:
There's `link_ignore_list` that uses globbing instead
`link_ignore_list = https://github.com/**`

Reply:
^ Optional.